lib/Doctrine/DBAL/DriverManager.php

```diff
@@ -142,17 +142,29 @@ public static function getConnection(
 
         $params = self::parseDatabaseUrl($params);
 
-        // URL support for MasterSlaveConnection
+        // @todo: deprecated, notice thrown by connection constructor
```

Comment:
Will the `Connection` constructor get a deprecation message in this PR?

Reply:
Ah right that's a part I forgot to backport from #4054

Reply:
Unsure about the `@todo` part, I copied it from #4054, is this a way we have to remember to remove things on 3.0.x?

Reply:
I don't know. I thought `@deprecated` and/or `trigger_error()` is usually in use.